### PR TITLE
removed supervisord in favor of systemd

### PR DESCRIPTION
**Pull request by bhand-mm**

### CODE REVIEW

File: .grim_build.sh [hunk 1]
```diff
@@ -10,6 +10,10 @@ export PATH="$GOPATH/bin:$PATH"
 
 cd "./$CLONE_PATH"
 
+go get github.com/bhand-mm/aws-sdk-go
```
Comment (kklipsch) [inline .grim_build.sh]: HAHAHAHAHAHAHAHHAHAHA

**Comment by kklipsch** [comment]
:+1:

**Comment by logie17** [comment]
Just curious, why?

**Comment by bhand-mm** [comment]
Mainly these but I could go on for a while about how impressed I am with systemd :-) :

1. systemd is much more powerful (you can run daemons as dependencies of others, when a file exists, when a socket is connected to, etc)
2. systemd is already installed by default on debian 8
3. supervisord is very out of date in the debian repos

**Pull request merged**